requests/pulls and committs.py

```diff
@@ -0,0 +1,62 @@
+import requests
```

Comment:
Pull Request prefix must contain prefix in ['GENERATOR', 'LEETCODE', 'HEXNUMBER', 'ITERATOR', 'TRIANGLE']